turek-hron-fsi3/solid-fenics/requirements.txt

```
@@ -0,0 +1,11 @@
+fenicsprecice==2.2.0rc1
```

```suggestion
fenicsprecice>=2.2
```
When `v2.2.0` is released.